sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -1471,6 +1523,12 @@ public partial class IisLogsDataSource
         public string Name { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> Streams { get { throw null; } }
     }
+    public partial class IngestionSettings
```

Comment:
This name is quite generic, could we be more specific on its name?

Reply:
Updated to MonitorWorkspaceIngestionSettings